web/app/routes/index.js

```diff
@@ -3,8 +3,21 @@ var router = express.Router();
 
 // frontend routes =========================================================
 // route to handle all angular requests
-router.get("*", function(req, res) {
-	res.sendfile("./public/index.html");
+var staticFilesLocation = process.env.NODE_ENV === "production" ? "dist" : "public";
+router.get("/login", function(req, res) {
+	if(req.isAuthenticated()) {
```

Comment:
This hasn't added it to the endpoints yet, only the frontend lol. We need to put the auth on the endpoints pretty much manually, tho, it should be easy